src/api.rs

```diff
@@ -7,6 +7,9 @@ const TRADING_POST_SALES_COMMISSION: i32 = 15; // %
 pub fn apply_trading_post_sales_commission(v: i32) -> Rational32 {
     Rational32::new(100 - TRADING_POST_SALES_COMMISSION, 100) * v
 }
+pub fn trading_post_price_for_revenue(v: Rational32) -> i32 {
```

Comment:
This is the inverse of `apply_trading_post_sales_commission`, so maybe it could be called something like `unapply_trading_post_sales_commission`.

Reply:
Renamed both to `{subtract/add}_trading_post_sales_commission` so they pair better.